pkg/core/native/policy.go

```diff
@@ -226,7 +228,10 @@ func (p *Policy) GetFeePerByteInternal(dao dao.DAO) int64 {
 
 // GetMaxVerificationGas returns maximum gas allowed to be burned during verificaion.
 func (p *Policy) GetMaxVerificationGas(_ dao.DAO) int64 {
-	return p.maxVerificationGas
+	if p.isValid {
+		return p.maxVerificationGas
+	}
+	return defaultMaxVerificationGas
```

Comment:
I don't really like that `maxVerificationGas` is a part of native Policy contract because of the way this variable is processed. We do not truly *store* this value in the chain file and cannot modify it via invoking native policy method, so `maxVerificationGas` behaves like a simple constant. I would suggest to store it and make getter and setter native policy methods, but it's not possible because we are restricted by C# node's native policy. The other option is to remove `maxVerificationGas` from native policy and make it just a constant.

Reply:
I hope it'll be a proper policy value one day, so let it stay the way it is now.